doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -200,6 +200,20 @@ statement it will overwrite the included file. Therefor any include
 statement at the end of the document will overwrite the already
 configured sections.
 
+Global Memcap
+-------------
+
+It represents the maximum amount of memory that can be used
```

Comment:
Suggest: "The global memcap value, if enabled, sets an upper bound on the sum of all configured memcap values. If the global memcap value is exceeded, Suricata will fail to start".